pom.xml

```diff
@@ -168,6 +168,8 @@
           </execution>
         </executions>
       </plugin>
+<!--
```

Comment:
@leesf FYI.. we are going to turn off spotless temporarily since the eclipse google style files have licensing conflicts.. Will do a follow on to course correct .

Reply:
OK